src/libs/httpserver/httpCookieManager.ts

```diff
@@ -137,6 +140,7 @@ export const resyncCookies = async (client: CozyClient): Promise<void> => {
 
   const stackCookie = cookies[appUrl]
 
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```

Comment:
Definitely, but I wanted this PR to be merged as fast as possible to avoid massive conflicts. Is it ok for you if we add an issue to handle these disables? And put it in the github dashboard

Reply:
Ok for me